src/routes/home/components/CreateGameDialog.vue

```diff
@@ -30,13 +30,17 @@
         {{ t('home.playAiContent2') }}
       </h4>
       <form name="create_game_form" class="d-flex align-center">
+        <StatsScoringDialog
+          activator-color="surface-2"
+          :show-button-text="false"
+        />
         <v-switch
           v-model="isRanked"
           :label="isRanked ? t('global.ranked') : t('global.casual')"
           data-cy="create-game-ranked-switch"
           color="surface-2"
+          class="d-flex align-center"
```

Comment:
align-center is for the flexbox container, not the child element, so this shouldn't be needed. The problem with the vertical alignment before was the bottom margin applied by the `class="mb-2"` on the other element. Removing that as you have should fix the vertical alignment without this line here

```suggestion
```